R/transformCounts.R

```diff
@@ -254,7 +263,17 @@ setMethod("transformFeatures", signature = c(x = "SummarizedExperiment"),
             stop("'name' must be a non-empty single character value and be ",
                  "different from `abund_values`.",
                  call. = FALSE)
-            }
+        }
+        # Check method
+        # If method is not single string, user has not specified transform method,
+        # or has given e.g. a vector
+        if(!.is_non_empty_string(method)){
+          stop("'method' must be a non-empty single character value. \n",
+               "Give one method from the following list: \n",
+               "log10', 'pa', 'z'",
```

Comment:
See above. It is essentially a crutch and therefore should be removed. The message "Have a look at the man page." would be better if you want to provided resource information

Reply:
Changed